rca/editorial/models.py

```diff
@@ -0,0 +1,145 @@
+
+from rca.utils.models import BasePage, RelatedPage
+
+
+class EditorialPageRelatedSchoolsAndResearchPages(RelatedPage):
+    source_page = ParentalKey(
+        "EditorialPage", related_name="related_schools_and_research_pages"
+    )
+    panels = [
+        PageChooserPanel("page", ["schools.SchoolPage", "research.ResearchCentrePage"])
+    ]
+
+
+class Author(models.Model):
+    name = models.CharField(max_length=128)
+
+    def __str__(self):
+        return self.name
+
+
+class EditorialPageArea(models.Model):
+    page = ParentalKey("EditorialPage", related_name="areas")
+    area = models.ForeignKey(
+        "people.AreaOfExpertise", related_name="editorial", on_delete=models.CASCADE
+    )
+    panels = [FieldPanel("area")]
+
+    def __str__(self):
+        return self.area.title
+
+
+class EditorialPage(BasePage):
+    template = "patterns/pages/editorial/editorial_detail.html"
+    introduction = models.CharField(blank=True, max_length=255)
+    hero_image = models.ForeignKey(
+        "images.CustomImage",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+    video = models.URLField(blank=True)
+    video_caption = models.CharField(
+        blank=True,
+        max_length=80,
+        help_text="The text displayed next to the video play button",
+    )
+    introduction_image = models.ForeignKey(
+        "images.CustomImage",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+
+    author = models.ForeignKey(
+        "editorial.Author",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+    published_at = models.DateField()
+    contact_email = models.EmailField(blank=True, max_length=254)
+
+    body = StreamField(
+        [
+            ("heading", blocks.CharBlock()),
+            ("paragraph", blocks.RichTextBlock()),
+            ("image", ImageChooserBlock()),
+            ("embed", EmbedBlock()),
+        ],
+        blank=True,
+    )
+
+    content_panels = BasePage.content_panels + [
+        FieldPanel("introduction"),
+        ImageChooserPanel("hero_image"),
+        MultiFieldPanel(
+            [
+                FieldPanel("video"),
+                FieldPanel("video_caption"),
+                ImageChooserPanel("introduction_image"),
+            ],
+            heading="Introductory Video",
+        ),
+    ]
+
+    key_details_panels = [
+        FieldPanel("published_at"),
+        MultiFieldPanel(
+            [
+                InlinePanel(
+                    "related_schools_and_research_pages",
+                    label="School or Research Centre",
+                ),
+                InlinePanel("areas", label="Area"),
+            ],
+            heading="Related School, Research Centre or Area",
+        ),
+        FieldPanel("author"),
+        FieldPanel("contact_email"),
+        StreamFieldPanel("body"),
```

Comment:
**Blocking**
I think this needs to be in the content_panels section so it appears in the right section in the back end